pkgs/development/python-modules/salib/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchPypi,
+  python3Packages,
+}:
+let
+  finalAttrs = {
+    pname = "salib";
+    version = "1.5.1";
+    pyproject = true;
+
+    src = fetchPypi {
+      inherit (finalAttrs) pname version;
+      hash = "sha256-5KnDGbjdAplajcmD9XxFLLflttvUPnt4VskMtqMyu18=";
+    };
+
+    buildInputs = with python3Packages; [
+      hatchling
+      hatch-vcs
+    ];
+
+    build-system = with python3Packages; [ setuptools ];
+
+    dependencies = with python3Packages; [
+      numpy
+      scipy
+      matplotlib
+      pandas
+      multiprocess
+      pathos
+    ];
```

```suggestion
    ];
    
    optional-dependencies = {
      distributed = [ pathos ];
    };
```
https://github.com/SALib/SALib/blob/6fb0a9c144692be5993e50dc99575eab44c192be/pyproject.toml#L42-L45